app/adapters/version.js

```diff
@@ -6,4 +6,12 @@ export default class VersionAdapter extends ApplicationAdapter {
     let num = snapshot.record.num;
     return `/${this.namespace}/crates/${crateName}/${num}`;
   }
+
+  queryRecord(_store, _type, query) {
+    let { name, num } = query ?? {};
+    let baseUrl = this.buildURL('crate', name);
+    let url = `${baseUrl}/${num}`;
+    let data = { ...query, name: undefined, num: undefined };
+    return this.ajax(url, 'GET', { data });
+  }
```

Comment:
wouldn't is be easier to override `urlForQueryRecord()` instead?

Reply:
Yeah, I've started with something like the following:
```javascript
  urlForQueryRecord(query) {
    let { name, num } = query ?? {};
    let baseUrl = this.buildURL('crate', name);
    let url = `${baseUrl}/${num}`;
    // The following used to remove them from URL's query string.
    delete query.name;
    delete query.num;
    return url
  }

```

I'm fine with both approaches. Do you have any opinions or preferences between them?

Reply:
Yeah, fair enough. I'll resolve the conflict and make the change then!

Reply:
> I'll resolve the conflict and make the change then

was that change lost in the rebase somehow? 😅

Reply:
oops, updated!